README.md

```diff
@@ -24,6 +24,15 @@ $ react-native run-android
 $ react-native run-ios
 ```
 
+## How to contribute?
+
+Install the pre-commit hook by running:
+```bash
+$ ln -s hooks/pre-commit.sh .git/hooks/pre-commit
```

Comment:
The link command is incorrect, it will hook the pre-commit hook to a non existing file.

You will need to use this (from the journey directory):
`ln -s ../../hooks/pre-commit.sh .git/hooks/pre-commit`

Once you have that the script should work.